src/components/structures/RoomView.js

```diff
@@ -469,7 +470,7 @@ module.exports = React.createClass({
         this.tabComplete.setCompletionList(
             MemberEntry.fromMemberList(members).concat(
                 CommandEntry.fromCommands(SlashCommands.getCommandList())
-            )
+            ).concat(EmojiEntry.entries)
```

Comment:
In my tests it worked well enough (that I actually used it myself for quite a while), but if we notice issues we can definitely try to be smarter about it. That it's at the end of the list of autocompletions definitely helps.

imo, the whole autocompletion thing needs to be changed so it isn't one giant list, but that's going to be a much larger change :sweat_smile: